Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -302,10 +302,10 @@ RefPtr<ImageBuffer> RemoteRenderingBackend::allocateImageBuffer(const FloatSize&
 }
 
 
-void RemoteRenderingBackend::createImageBuffer(const FloatSize& logicalSize, RenderingMode renderingMode, RenderingPurpose purpose, float resolutionScale, const DestinationColorSpace& colorSpace, PixelFormat pixelFormat, RenderingResourceIdentifier imageBufferIdentifier)
+void RemoteRenderingBackend::createImageBuffer(const FloatSize& logicalSize, RenderingMode renderingMode, RenderingPurpose purpose, float resolutionScale, const DestinationColorSpace& colorSpace, ImageBufferPixelFormat imageBufferPixelFormat, RenderingResourceIdentifier imageBufferIdentifier)
 {
     assertIsCurrent(workQueue());
-    RefPtr<ImageBuffer> imageBuffer = allocateImageBuffer(logicalSize, renderingMode, purpose, resolutionScale, colorSpace, pixelFormat, { }, imageBufferIdentifier);
+    RefPtr<ImageBuffer> imageBuffer = allocateImageBuffer(logicalSize, renderingMode, purpose, resolutionScale, colorSpace, convertToPixelFormat(imageBufferPixelFormat), { }, imageBufferIdentifier);
```

Comment:
This is confusing. 

1. Here `convertToPixelFormat(imageBufferPixelFormat)` is called because `allocateImageBuffer()` takes `PixelFormat`
2. It passes it to `allocateImageBufferInternal()` which passes it to `ImageBuffer::create()`
3. `ImageBuffer::create()` calls `convertToImageBufferPixelFormat(pixelFormat)` because `ImageBufferParameters` now has `ImageBufferPixelFormat`

So we did the conversion twice. Why do not we make everything `ImageBufferPixelFormat`? Only when we want a PixelBuffer we can call `convertToPixelFormat(imageBufferPixelFormat)`.